features/related-website-sets.yml

```diff
@@ -0,0 +1,15 @@
+name: Related website sets
+description: Related website sets declares relationships between sites so the browser allows cookie and storage access between them through methods such as `document.requestStorageAccessFor()`.
+spec:
+  - https://wicg.github.io/first-party-sets/
+  - https://privacycg.github.io/requestStorageAccessFor/
+discouraged:
+  removal_date: 2026-06-01 # Chrome 150
+  reason: Due to declining interest from web developers and lack of interest from other browser vendors, Google decided to withdraw many Privacy Sandbox features including related website sets.
```

Comment:
See comment about Attribution Reporting on how we phrase/frame this: https://github.com/web-platform-dx/web-features/pull/3526/files#r2627991718.